### DISCUSSION TITLE
Is it possible to get multiple account addresses ?

### DISCUSSION THREAD

**Discussion by rikotacards** [wevm/wagmi#4173] [Q&A] [answered]
Using avalanche's core wallet, I have multiple accounts. Is it possible to get the accounts? Right now, `useAccount`, just gets a single account. There is `account.addresses`, which always returns a list withe one address.

**Comment by tmm** [comment, marked as answer]
[`account.addresses`](https://wagmi.sh/vue/api/composables/useAccount#addresses) is what you want, but not all wallets expose multiple addresses. The only wallet that I know does this at the moment is MetaMask.

https://github.com/user-attachments/assets/cf84d0af-8401-498e-845a-e3e821d5e20d

**Reply by rikotacards** [reply]
Thanks! Ok, so it depends on the wallet. Didn't realize that. Somehow the avalanche core wallet on web is able to detect addresses. They must have some special API.
